app/serializers/fixed_stream_serializer.rb

```diff
@@ -12,6 +12,7 @@ def call(stream:, measurements:, stream_daily_averages:)
 
   def serialized_stream(stream)
     thresolds = Stream.thresholds(stream.sensor_name, stream.unit_symbol)
+    update_frequency = stream.session.user.username == 'US EPA AirNow' ? '1 hour' : '1 minute'
```

Comment:
There is `delegate` used in the Session class:
https://github.com/HabitatMap/AirCasting/blob/294f1969339a6310af7084995757e7f25b358fe9/app/models/session.rb#L25

Thanks to delegation we can call `username` directly on a session object, so
`stream.session.user.username` can be shorten to `stream.session.username`.